Packs/Carbon_Black_Enterprise_Response/Integrations/CarbonBlackResponseV2/CarbonBlackResponseV2.py

```diff
@@ -861,6 +861,15 @@ def fetch_incidents(client: Client, max_results: int, last_run: dict, first_fetc
         if not alert_id or not alert_name:
             demisto.debug(f'{INTEGRATION_NAME} - Alert details are missing. {str(alert)}')
 
+        if ioc_attr := alert.get('ioc_attr'):
+            try:
+                alert['ioc_attr'] = json.loads(ioc_attr)
+                highlights = alert['ioc_attr'].get('highlights', {})
+                for i, attribute in enumerate(highlights):
+                    highlights[i] = attribute.replace("PREPREPRE", "").replace("POSTPOSTPOST", "")
```

Comment:
```suggestion
                    highlights[i] = attribute.lstrip("PREPREPRE").lstrip("POSTPOSTPOST")
```

Reply:
It's not always a prefix, I have seen cases that PREPREPRE is in the middle of the string for example cmd.exe /cPREPREPRE/x